flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolBuilder.java

```diff
@@ -54,23 +56,47 @@ public SlotPoolBuilder setBatchSlotTimeout(Time batchSlotTimeout) {
 		return this;
 	}
 
+	public SlotPoolBuilder setIdleSlotTimeout(Time idleSlotTimeout) {
+		this.idleSlotTimeout = idleSlotTimeout;
+		return this;
+	}
+
 	public SlotPoolBuilder setClock(Clock clock) {
 		this.clock = clock;
 		return this;
 	}
 
-	public TestingSlotPoolImpl build() throws Exception {
+	public TestingSlotPoolImpl build(JobID jobID, Boolean connectToResourceManager) throws Exception {
```

Comment:
Boolean -> boolean

Reply:
Can we add setters for `jobID` and `connectToResourceManager`, and not make them params of `build()`?
Usually `build()` params are necessary ones without default values.